keyboards/pauperboards/brick/keymaps/default/keymap.c

```diff
@@ -1,19 +1,3 @@
-/* Copyright 2023 Juno Nguyen <juno.ngx@gmail.com>
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 2 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
 #include QMK_KEYBOARD_H
```

Comment:
Must have GPL compliant headers (which you have removed?)

Reply:
This was accidental, and will be fixed in the next push.

Reply:
I restored my json implementation.